contracts/BoostedEthBridgeSwapper.sol

```diff
@@ -88,4 +86,12 @@ contract BoostedEthBridgeSwapper is ZkSyncBridgeSwapper {
     function yvCrvStEthPerEth() public view returns (uint256) {
         return (1 ether ** 2) / ethPerYvCrvStEth();
     }
+
+    function tokens(uint256 index) external view returns (address) {
```

Comment:
Same as for `GroGvtBridgeSwapper`, I find this unclean as index must be 0 or 1 or exchange will revert while here we accept any value > 0 for `yvCrvStEth`.